### DISCUSSION TITLE
General discussion about AntiMicroX and further development

### DISCUSSION THREAD

**Discussion by pktiuk** [AntiMicroX/antimicrox#215] [General]
### About
Issue for general discussion about AntiMicroX ant its further development.  
Discussion about current state of app and its further development in general.

### Current plans
This fork is a rather new, that's why we are currently focused on cleaning up the repository and preparing it for further development.  
We are also working on making AntiMicroX easily available and updateable for users.  
Then we are going to move on to fixing existing bugs
Later we will think about possible improvements and porting to QT6 (not released yet)

**Offtopic comments will be deleted/hidden to avoid spam.

**Comment by gombosg** [comment]
I totally agree about this. This is a feature-packed, well-functioning piece of software, ridden with bugs and codebase quality issues due to the long chain of maintainers.

IMO general priorities should be, considering our limited volunteer efforts as a bottleneck:
1. Keep the builds green!
1. Fix bugs for existing users, especially visible and ugly ones affecting basic functionality (like #6 and #7 😛).
1. Reach a wider audience within existing OSes via consistent releases and more packaging (#2, #9, #32)
1. Understand code base, and clean up. Remove dead code, refactor variable names, reformat syntax. This could open the project up for other people's PR submissions, too!
1. Add more OS support (#4, possibly also OSX though that may be a long shot) to reach even more users.
1. IMO the lowest priority is adding new features, because adding new features is only worth the effort if we already have a considerable amount of users (unless it's some 'killer feature') and the codebase quality is good enough to make the feature easier to maintain.

**Comment by mhalano** [comment]
I would like to add my own issue to this discussion because it's quite aligned. The issue #100 I opened is to more Linux distros can use AntiMicroX. I saw a good discussion was started on #87 so antimicro becomes antimicrox on Debian and Ubuntu repositories.
